README.md

```diff
@@ -1,2 +1,151 @@
+</ul>
+
+### Key Features <a name="key-features"></a>
+
+- **Add and remove tasks**
+
+- **JavaScript Classes**
+
+  - The tasks are managed and manipulated using classes and helper functions within those classes.
+
+- **Store and retrieve data**
+
+  - The displayed tasks are stored and retrieved from the local storage of the browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser (e.g. Firefox)
+- An IDE (e.g. [Vususal Studio Code](https://code.visualstudio.com/download))
+- [npm](https://nodejs.org/en/)
+- Terminal
+
+### Setup
+
+To clone this repository to your desired folder, follow the steps below:
+
+**Using the command line**
+
+- Use the following commands
+
+```sh
+  cd my-folder
+  git clone git@github.com:mohisa302/Leader-board.git
+```
+
+**Using GitHub Desktop app**
+
+- Follow the steps below
+
+  - Visit this link "git@github.com:mohisa302/Leader-board.git"
+  - Click the green button labelled "code"
+  - Select the "Open with GitHub Desktop" option
+  - After the GitHub Desktop add opens, click the "clone repo" button
+
+### Usage
+
+To run the project, execute the following command:
+
+**Click the "Go Live" button on your IDE(VS Code in this case)**
+
+### Deployment
+
+You can deploy this project by:
+
+- visiting your repository on [GitHub](https://github.com)
+- Click on settings
+- Go to the "Pages" option
+- Select the main branch and save
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mohi**
+
+- GitHub: [@mohisa302](https://github.com/mohisa302)
+- Twitter: [@Kholochelam](https://twitter.com/Kholochelam)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mohadese-sadeghi-692551199/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[will improve page elements]**
+- [ ] **[will improve styles]**
+- [ ] **[will improve js functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a ⭐️ and let me know what you like in particular.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the whole Microverse community for their help and contributions towards this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- **[OPTIONAL]** If the name of your LICENSE file is LICENSE.md, please 🙏🏽 correct the link path and make sure that the link works as expected. To have a professional README file.

Reply:
done